Sources/Datadog/Datadog.swift

```diff
@@ -340,6 +340,10 @@ public class Datadog {
         // Deinitialize `Datadog`:
         defaultDatadogCore = NOOPDatadogCore()
     }
+
+    // MARK: - Internal Proxy - exposure of internal classes (Mostly used for cross platform libraries)
+
+    public private(set) static var _internal = _InternalProxy()
```

Comment:
This could be:
```suggestion
    public internal(set) static var _internal = _InternalProxy()
```

So you won't need to override it in `RUMMonitor`, WDYT?

Reply:
I'm not sure I understand what you're suggesting? The variable in RUMMonitor is a completely different proxy.

Reply:
oops sorry, that's the wrong line!